src/open_inwoner/templates/pages/contactmoment/detail.html

```diff
@@ -1,21 +1,42 @@
 {% extends 'master.html' %}
-{% load i18n anchor_menu_tags card_tags file_tags grid_tags table_tags solo_tags form_tags button_tags %}
+{% load i18n grid_tags table_tags button_tags icon_tags %}
 
 {% block content %}
     {% if contactmoment %}
         {% render_grid %}
             {% render_column span=12 %}
-                {# Contactmoment/dashboard. #}
-                <h1 class="h1" id="title">{{ page_title }}</h1>
+                <h1 class="h1" id="title">{{ contactmoment.onderwerp }}</h1>
                 {% include "components/Dashboard/Dashboard.html" with metrics=metrics only %}
             {% endrender_column %}
-            {% render_column start=4 span=6 %}
+        {% endrender_grid %}
 
-                <div class="contactmoment__details">
-                {% contactmoment_table contactmoment zaak=zaak %}
+        <div class="container--no-margin">
+            <div class="contactmoment">
+                <h3>{% trans "Uw vraag" %}</h3>
```

Comment:
Do not jump from H1 to H3 without using a H2 Tag...! (bad for the hierarchy in accessibility software).
Also looking at the design you can see all H3's in your template need to actually be H2's (but... with a font-size of 20px) - you could use the "h2" class here as a basis, and/or create a special H2 heading within Contactmoment SCSS. Something like ".contactmoment__heading-2" but then that one would need to be a Flexbox because of the date field.

Reply:
Fortunately, we're not displaying the response date, so this complication doesn't arise.

Reply:
Alright, but you will still need to use an H2 _tag_ here instead of an H3 _tag_.